### PR TITLE
fix: add singularAlias param on associations hasMany

### PR DESCRIPTION
**Pull request by arodu**
in template view, when association is hasMany, I can't get association name in singular

### CODE REVIEW

File: templates/bake/Template/view.twig [hunk 1]
```diff
@@ -118,7 +118,7 @@
 {% endif %}
 {% set relations = associations.BelongsToMany|merge(associations.HasMany) %}
 {% for alias, details in relations %}
-{% set otherSingularVar = alias|variable %}
+{% set otherSingularVar = details.singular|variable %}
```
Comment (ADmad) [inline templates/bake/Template/view.twig]: Why is a new `singular` key needed? You can directly do `alias|singularize|variable`.

Comment (arodu) [reply templates/bake/Template/view.twig]: oh.. that's cool!

Comment (dereuromark) [reply templates/bake/Template/view.twig]: ping @arodu 
You want to finish this up for merge?

Comment (arodu) [reply templates/bake/Template/view.twig]: Hi.. @dereuromark 
So... a new singular key is not needed? 
but I can still fix templates/bake/Template/view.twig?

Comment (ADmad) [reply templates/bake/Template/view.twig]: Yes, just update the template

**Comment by arodu** [comment]
@ADmad done!

**Review by dereuromark — APPROVED** [review]
Thank you :)

**Pull request merged**